documentation/Extensions.md

```diff
@@ -98,3 +98,10 @@ Extensions and legacy features deliberately not supported
 * Use of host FORMAT labels in internal subprograms (PGI-only feature)
 * ALLOCATE(TYPE(derived)::...) as variant of correct ALLOCATE(derived::...) (PGI only)
 * USE association of a procedure interface within that same procedure's definition
+* After "TYPE,EXTENDS(T1)::T2;...", the nonstandard structure constructor
+  T2(T1(x)) is accepted by PGI/GNU/Intel.  Use T2(T1=T1(x)) or T2(x) instead.
```

Comment:
Should this line also begin with `*`?

Reply:
The leading asterisk in markdown indicates a new bullet point, which would not look good in the middle of a sentence.